src/App.jsx

```diff
@@ -14,33 +16,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'string';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
Why not use enum here? That will make the code more readable by showing all possible options in one place.

BTW - using TypeScript is even a better idea :)